vassal-app/src/main/java/VASSAL/tools/imports/adc2/ADC2Module.java

```diff
@@ -712,15 +713,6 @@ public String getClassName() {
       return pieceClass.getName();
     }
 
-    protected void setValue(int index, String value) {
-      byte[] b = value.getBytes();
-      int result = 0;
-      for (int i = 0; i < 4; ++i)
-        result = (result << 8) + (b[i] & 0xff);
-      values[index] = result;
-      types[index] = ValueType.TEXT;
-    }
-
```

Comment:
On reflection, this it's useful to have this here as it describes the file format, in the absence of a document.

The bytes should be ASCII.